changelog/README.rst

```diff
@@ -0,0 +1,43 @@
+:orphan:
+
+Changelog
+=========
+
+This directory contains "news fragments" which are short files that contain a
+small **ReST**-formatted text that will be added to the next what's new page.
+
+Make sure to use full sentences with correct case and punctuation, and please
+try to use Sphinx intersphinx using backticks. The fragment should have a
+header line and an underline using ``------`` 
+
+Each file should be named like ``<PULL REQUEST>.<TYPE>.rst``, where
+``<PULL REQUEST>`` is a pull request number, and ``<TYPE>`` is one of:
+
+* ``new_function``: New user facing functions.
+* ``deprecation``: Changes existing code to emit a DeprecationWarning.
+* ``future``: Changes existing code to emit a FutureWarning.
+* ``expired``: Removal of a deprecated part of the API.
+* ``compatibility``: A change which requires users to change code and is not
+  backwards compatible. (Not to be used for removal of deprecated features.)
+* ``c_api``: Changes in the Numpy C-API exported functions
+* ``new_features``: New user facing features like ``kwargs``.
+* ``improvements``: Performance and edge-case changes
+* ``changes``: Other changes
+
+So for example: ``123.c_api.rst``, ``456.new_feature.rst``.
+
+If you are unsure what pull request type to use, don't hesitate to ask in your
+PR.
+
+Note that the ``towncrier`` tool will automatically reflow your text, so it
```

Comment:
I don't think this is true for our template, so this should probably be removed  -- you removed the automatic insertion of list items from the pytest template.

Reply:
removing